common/src/main/java/com/sequenceiq/cloudbreak/telemetry/databus/AbstractDatabusStreamConfiguration.java

```diff
@@ -3,7 +3,9 @@
 import java.util.HashMap;
 import java.util.Map;
 
-public abstract class AbstractDatabusStreamConfiguration {
+import com.sequenceiq.cloudbreak.telemetry.streaming.CommonStreamingConfiguration;
+
+public abstract class AbstractDatabusStreamConfiguration extends CommonStreamingConfiguration {
```

Comment:
@szabolcs-horvath As discussed offline, please make sure that the child classes of this class overwrite the isStreamingEnabled() function, because by default that function returns false in CommonStreamingConfiguration.